providers/js/fido-u2f-admin.js

```diff
@@ -9,29 +9,30 @@
 			$button.addClass( 'clicked' );
 		}
 
-		setTimeout( function() {
-			window.console.log( 'sign', u2fL10n.register.request );
+		window.console.log( 'sign', u2fL10n.register.request );
 
-			$button.text( u2fL10n.text.insert ).append( '<span class="spinner is-active" />' );
+		$button.text( u2fL10n.text.insert ).append( '<span class="spinner is-active" />' );
 
-			$( '.spinner.is-active', $button ).css( 'margin', '2.5px 0px 0px 5px' );
+		$( '.spinner.is-active', $button ).css( 'margin', '2.5px 0px 0px 5px' );
 
-			u2f.register( [ u2fL10n.register.request ], u2fL10n.register.sigs, function( data ) {
-				window.console.log( 'Register callback', data, this );
+		var registerRequest = {
```

Comment:
The `registerRequest` doesn't include the `appId` in the latest version of the JS API.